charmdir.go

```diff
@@ -246,6 +248,32 @@ func (dir *CharmDir) LXDProfile() *LXDProfile {
 	return dir.lxdProfile
 }
 
+// Manifest returns the Manifest representing the manifest.yaml file
+// for the charm expanded in dir.
+func (dir *CharmDir) Manifest() *Manifest {
+	return dir.manifest
+}
+
+// ComputedSeries of a charm. This is to support legacy logic on new
+// charms that use Systems.
+func (dir *CharmDir) ComputedSeries() []string {
+	if len(dir.manifest.Bases) == 0 {
+		return dir.meta.Series
+	}
+	// The slice must be ordered based on system appearance but
+	// have unique elements.
+	seriesSlice := []string(nil)
+	seriesSet := set.NewStrings()
+	for _, base := range dir.manifest.Bases {
+		series := base.String()
```

Comment:
I don't like the fact that a stringified base is a series. I will look at that when I bring base in from systems.